extensions/sql-database-projects/src/models/projectEntry.ts

```diff
@@ -73,18 +73,15 @@ export class DacpacReferenceProjectEntry extends FileProjectEntry implements IDa
 }
 
 export class SystemDatabaseReferenceProjectEntry extends FileProjectEntry implements IDatabaseReferenceProjectEntry {
-	constructor(uri: Uri, public ssdtUri: Uri, public databaseName: string, public databaseVariableLiteralValue: string | undefined, public suppressMissingDependenciesErrors: boolean) {
-		super(uri, '', EntryType.DatabaseReference);
+	constructor(public databaseName: string, public databaseVariableLiteralValue: string | undefined, public suppressMissingDependenciesErrors: boolean) {
+		super(Uri.file(databaseName), databaseName, EntryType.DatabaseReference);
 	}
 
+	/**
+	 * Returns the name of the system database - this is used for deleting the system database reference
+	 */
 	public override pathForSqlProj(): string {
-		// need to remove the leading slash for system database path for build to work on Windows
-		return utils.convertSlashesForSqlProj(this.fsUri.path.substring(1));
-	}
-
-	public ssdtPathForSqlProj(): string {
-		// need to remove the leading slash for system database path for build to work on Windows
-		return utils.convertSlashesForSqlProj(this.ssdtUri.path.substring(1));
+		return this.databaseName;
```

Comment:
`deleteDatabaseReference()` takes the name of the db ref to delete, which for dacpac and project references is the `pathForSqlProj()`. For system dacpacs, it takes the name (Master or MSDB), so for simplicity I changed this to return the database name since the actual uris aren't needed here now that DacFx handles adding and removing these. Think that's ok or should I add a check for system db refs in the delete code in project.ts?